modules/openfast-library/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ target_link_libraries(openfast_prelib
   icedynlib
   icefloelib
   ifwlib
+  lindynlib
```

Comment:
If `LynDyn` is only used in the UA driver, we don't need to include it in the openfast-library.  I noticed it is not included in the CMakeLists.txt for Simulink or in the FAST VS projects.

Reply:
resolved with 1dd60b96d